site-src/guides/serve-multiple-lora-adapters.md

```diff
@@ -1,11 +1,11 @@
 # Serve LoRA adapters on a shared pool
 A company wants to serve LLMs for document analysis and focuses on audiences in multiple languages, such as English and Spanish.
 They have a fine-tuned LoRA adapter for each language, but need to efficiently use their GPU and TPU capacity.
-You can use Gateway API Inference Extension to deploy dynamic LoRA fine-tuned adapters for each language (for example, `english-bot` and `spanish-bot`) on a common base model and accelerator.
+You can use Inference Gateway to deploy dynamic LoRA fine-tuned adapters for each language (for example, `english-bot` and `spanish-bot`) on a common base model and accelerator.
```

Comment:
s/You can use Inference Gateway/You can use an Inference Gateway/

The docs should use 'an Inference Gateway' throughout instead of 'Inference Gateway'. Do you agree @robscott?